src/Parser/Core/Modules/Features/SpellInfo.js

```diff
@@ -26,6 +26,7 @@ export default {
     int: false,
     crit: true,
     hasteHpct: false, // static CD
+    hasteHpm: true,
```

Comment:
Just making sure: are you sure about this? It seems strange

Reply:
Pretty sure. Tick rate should be every 3 s: https://www.warcraftlogs.com/reports/f7Gg2HRbwYPDqJ1B#fight=70&type=healing&source=241&ability=242622&view=events

Also verified here https://questionablyepic.com/restodruid-trinkets/
![img](https://i.gyazo.com/f72eecc1c14f4377615a8fe96fe1b922.png)